daemon/rpc/handlers.go

```diff
@@ -238,7 +238,10 @@ func (server *Server) SwapOut(ctx context.Context, req *SwapOutRequest) (*SwapOu
 
 	log.Info("Swap created: ", swap.SwapId)
 
-	return &SwapOutResponse{}, nil
+	return &SwapOutResponse{
+		SwapId:     swap.SwapId,
+		AmountSats: uint64(swap.InputAmount.Mul(decimal.NewFromInt(1e8)).IntPart()), // nolint:gosec
```

Comment:
should we add `Mul` to the money package?

Reply:
good point, will do